projects/homework 4/binary_representation/binaryToDecimal.c

```diff
@@ -0,0 +1,17 @@
+#include "binaryToDecimal.h"
+#include <math.h>
+
+#define SIZE 8
```

Comment:
Хотелось работать с настоящими int-ами, так что правильнее так:
```suggestion
#define SIZE (sizeof(int) * 8)
```